src/styles/1c-light.css

```diff
@@ -0,0 +1,133 @@
+/*!
+  Theme: 1c-light
+  Description: Style IDE 1C:Enterprise 8
+  Author: (c) Barilko Vitaliy <barilkovetal@gmail.com>
+  Maintainer: @Diversus23
+  Website: https://softonit.ru/
+  License: see project LICENSE
+  Touched: 2023
+*/
+
+pre code.hljs {
+    display: block;
+    overflow-x: auto;
+    padding: 1em;
+  }
```

Comment:
Removed, this is added by the build process.
```suggestion
```

Reply:
Thanks for the review. Done